adminSiteServer/apiRoutes/charts.ts

```diff
@@ -717,7 +717,7 @@ export async function createChart(
 
         return { success: true, chartId: chartId }
     } catch (err) {
-        return { success: false, error: String(err) }
+        return { success: false, error: { message: String(err), status: 500 } }
```

Comment:
Might be better to do:

```suggestion
        return { success: false, error: new JsonError(err) }
```

It seems like an anti-pattern to me, but this is the way the admin was built 😅

Reply:
ah, that makes much more sense!

Reply:
The error message didn't make it through when I used JsonError. I just merged as is for now :)